cmd/kubeapps-apis/plugins/pkg/resourcerefs/resourcerefs_test.go

```diff
@@ -18,680 +18,19 @@ import (
-			ExistingReleases: []TestReleaseStub{
-				{
-					Name:      releaseName,
-					Namespace: releaseNamespace,
-					Manifest: `
----
-apiVersion: v1
-kind: List
-items:
-- apiVersion: apps/v1
-  kind: Deployment
-  metadata:
-    name: redis-test
-    namespace: default
-- apiVersion: v1
-  kind: Service
-  metadata:
-    name: redis-test
-    namespace: default
-`,
-				},
-			},
-			ExpectedResourceRefs: []*corev1.ResourceRef{
-				{
-					ApiVersion: "apps/v1",
-					Name:       "redis-test",
-					Namespace:  "default",
-					Kind:       "Deployment",
-				},
-				{
-					ApiVersion: "v1",
-					Name:       "redis-test",
-					Namespace:  "default",
-					Kind:       "Service",
-				},
-			},
-		},
-		{
-			Name: "renders a rolelist of items (2)",
-			// See https://kubernetes.io/docs/reference/kubernetes-api/authorization-resources/role-v1/#RoleList
-			ExistingReleases: []TestReleaseStub{
-				{
-					Name:      releaseName,
-					Namespace: releaseNamespace,
-					Manifest: `
----
-apiVersion: v1
-kind: RoleList
-items:
-- apiVersion: rbac.authorization.k8s.io/v1
-  kind: Role
-  metadata:
-    name: role-1
-    namespace: default
-- apiVersion: rbac.authorization.k8s.io/v1
-  kind: Role
-  metadata:
-    name: role-2
-    namespace: default
-`,
-				},
-			},
-			ExpectedResourceRefs: []*corev1.ResourceRef{
-				{
-					ApiVersion: "rbac.authorization.k8s.io/v1",
-					Name:       "role-1",
-					Namespace:  "default",
-					Kind:       "Role",
-				},
-				{
-					ApiVersion: "rbac.authorization.k8s.io/v1",
-					Name:       "role-2",
-					Namespace:  "default",
-					Kind:       "Role",
-				},
-			},
-		},
-		{
-			Name: "renders a ClusterRoleList of items (2)",
-			// See https://kubernetes.io/docs/reference/kubernetes-api/authorization-resources/cluster-role-v1/#ClusterRoleList
-			ExistingReleases: []TestReleaseStub{
-				{
-					Name:      releaseName,
-					Namespace: releaseNamespace,
-					Manifest: `
----
-apiVersion: v1
-kind: ClusterRoleList
-items:
-- apiVersion: rbac.authorization.k8s.io/v1
-  kind: ClusterRole
-  metadata:
-    name: clusterrole-1
-- apiVersion: rbac.authorization.k8s.io/v1
-  kind: ClusterRole
-  metadata:
-    name: clusterrole-2
-`,
-				},
-			},
-			ExpectedResourceRefs: []*corev1.ResourceRef{
-				{
-					ApiVersion: "rbac.authorization.k8s.io/v1",
-					Name:       "clusterrole-1",
-					Namespace:  "test",
-					Kind:       "ClusterRole",
-				},
-				{
-					ApiVersion: "rbac.authorization.k8s.io/v1",
-					Name:       "clusterrole-2",
-					Namespace:  "test",
-					Kind:       "ClusterRole",
-				},
-			},
-		},
-	}
-}
-
 func TestGetInstalledPackageResourceRefs(t *testing.T) {
 	ignoredFields := cmpopts.IgnoreUnexported(
 		corev1.ResourceRef{},
 	)
 
-	testCases := []TestCase{}
-	testCases = append(testCases, TestCases1...)
-	testCases = append(testCases, TestCases2...)
+	testCases := []resourcerefstest.TestCase{}
+	testCases = append(testCases, resourcerefstest.TestCases1...)
```

Comment:
Related to an earlier comment, my preference would be to just leave the tests here in this module (not re-using them in other places), and instead update other modules to only test their specific code in the unit tests there (given that this module tests a lot of the functionality already).

Reply:
see what you think about what I said earlier as far as the reasoning why I did that. In the end, I WILL do as you see fit.